### ISSUE TITLE
Tagging and adding lists in edit screen should be enhanced

### ISSUE THREAD

**Issue by mpcjanssen** [mpcjanssen/simpletask-android#132]
Currently the tagging and adding lists for tasks in the main screen is more useful than the similar functionality in the Edit task screen. For example you cannot easily delete tags in the edit screen.

The edit screen Add tag and Add list dialogs should mimic the main screen ones.

**Comment by smichel17** [comment]
@mpcjanssen I agree, there should be a shared interface for editing tasks between the main and edit screens. I am working on that now.. When you edit multiple tasks, how does Simpletask decide which order to display them in? Could you customize that, eg, so they always appear in the same order that they were when you selected them?

@alehandrof I agree with you in principle. In practice, I'm concerned that the animation time needed would make the app completely unusable. Beautiful, but still unusable.

**Comment by smichel17** [comment]
I think this issue can be closed when the issues that reference this one are closed.

**Comment by smichel17** [comment]
>In some sense, there shouldn't be Add Tag/List buttons on the main screen at all. There should be an Edit task button whose Tag/List buttons work like the ones that are currently on the main screen.

The alternative, which I am leaning towards, is that there should be no edit task button -- all editing should be done from the main screen, in a separate fragment but the same activity.
